data-prepper-plugins/cloudwatch-logs/README.md

```diff
@@ -0,0 +1,76 @@
+# CloudWatch Logs Sink
+
+This Data Prepper sink allows the sending of log data to CloudWatch Logs via a CloudWatchLogsClient.
+
+## Usages
+
+The CloudWatch Logs sink should be configured as part of Data Prepper pipeline yaml file.
+
+## Configuration Options
+
+```
+pipeline:
+  ...
+  sink:
+    - cloudwatch_logs:
+        aws:
+          region: us-east-1
+          sts_role_arn: arn:aws:iam::123456789012:role/Data-Prepper
+          sts_header_overrides:
```

Comment:
all have example data except this? why?

Reply:
Sorry about that. I missed those values in the original draft. I have added in example values now and another parameter that was missing.

Reply:
We probably don't even need to show this in the example. It is not commonly used.